src/app/components/event-page/event-page.component.ts

```diff
@@ -81,6 +96,41 @@ export class EventPageComponent implements OnInit {
 		this.tagsList = this.event.tags.split('*');
 	}
 
+	getCurrentParticipants(): void {
+		this.attendanceService.getParticipantsForEvent(this.eventId).subscribe((result) => {
+			this.participantsArray = result;
+			this.checkIfCurrentuserAttends();
+			this.checkIfMaximumReached();
+		})
+	}
+
+	checkIfCurrentuserAttends(): void {
+		this.isCurrentUserNotAttending = true;
+		this.participantsArray.forEach(element => {
+			if (element.userId == this.currentUserId) {
+				this.isCurrentUserNotAttending = false;
+			}
+		});
+	}
+
+	checkIfMaximumReached(): void {
+		if (this.participantsArray.length == this.event.maximumParticipants) {
```

Comment:
this.isMaximumReached = this.participantsArray.length == this.event.maximumParticipants